lib/rust/parser/jni/Cargo.toml

```diff
@@ -17,4 +17,4 @@ jni = "0.19.0"
 
 [lib]
 name = "enso_parser"
-crate-type = ["cdylib"]
+crate-type = ["cdylib", "staticlib"]
```

Comment:
This is a chance you suggested me to do yesterday in order to also get static library for the parser, right? I haven't tested whether it solves my problem (linking Native Image of Enso with the parser) yet.

Reply:
Oops, didn't mean to include that here!